diff --git a/node_modules/react-quill/lib/index.js b/node_modules/react-quill/lib/index.js
index 9b59628..f313f81 100644
--- a/node_modules/react-quill/lib/index.js
+++ b/node_modules/react-quill/lib/index.js
@@ -7,7 +7,7 @@
 var _react = _interopRequireDefault(require('react'));
 
 var _reactDom = _interopRequireDefault(require('react-dom'));
-
+var findDOMNode = function findDOMNode(instance) { return _reactDom.default.findDOMNode(instance); };
 var _quill = _interopRequireDefault(require('quill'));
 
 var _propTypes = _interopRequireDefault(require('prop-types'));
@@ -1071,7 +1071,7 @@
 	},
 
 	getEditor: function getEditor() {
-		return this.editor;
+		return this.quill;
 	},
 
 	getEditingArea: function getEditingArea() {
@@ -1080,7 +1080,7 @@
 
 	getEditingAreaElement: function getEditingAreaElement() {
 		var editingArea = this.getEditingArea();
-		return editingArea.firstChild;
+		return editingArea;
 	},
 
 	/*
