--- a/node_modules/react-quill/lib/index.js
+++ b/node_modules/react-quill/lib/index.js
@@ -493,7 +493,7 @@
 	},
 
 	getEditor: function getEditor() {
-		return this.editor;
+		return this.quill;
 	},
 
 	focus: function focus() {
@@ -501,7 +501,7 @@
 	},
 
 	blur: function blur() {
-		this.selection = this.getEditor().getSelection();
+		this.selection = this.quill.getSelection();
 		this.quill.blur();
 	},
 
@@ -510,7 +510,7 @@
 	},
 
 	getEditingArea: function getEditingArea() {
-		return _reactDom2.default.findDOMNode(this.editingArea);
+		return this.editingArea.getEditingArea();
 	},
 
 	/*
@@ -628,7 +628,7 @@
 			value: this.getEditor().getContents()
 		};
 		// Check for undefined class, and add the correct class name.
-		if (this.editingArea.className === 'ql-container ql-snow') {
+		if (this.editingArea.editingArea && this.editingArea.editingArea.className === 'ql-container ql-snow') {
 			this.editingArea.className = this.props.className ? this.props.className : '';
 		}
 	},
